tests/integration/test_remoted/test_configuration/test_basic_configuration_syslog_allowed_denied_ips_valid.py

```diff
@@ -56,7 +80,17 @@ def test_allowed_denied_ips_syslog(get_configuration, configure_environment, res
     requests.packages.urllib3.disable_warnings(category=InsecureRequestWarning)
     cfg = get_configuration['metadata']
 
-    log_callback = remote.callback_detect_syslog_allowed_ips(cfg['allowed-ips'])
+    address = cfg['allowed-ips'][:-3]
+    netmask = cfg['allowed-ips'][-3:]
+    allowed_ips = ipaddress.ip_address(address).exploded + netmask
+    log_callback = remote.callback_detect_syslog_allowed_ips(allowed_ips)
     wazuh_log_monitor.start(timeout=5, callback=log_callback, error_message="Wazuh remoted didn't start as expected.")
 
-    compare_config_api_response([cfg], 'remote')
+    if 'allowed-ips2' in cfg:
+        address2 = cfg['allowed-ips2'][:-3]
+        netmask2 = cfg['allowed-ips2'][-3:]
+        allowed_ips2 = ipaddress.ip_address(address2).exploded + netmask2
+        log_callback = remote.callback_detect_syslog_allowed_ips(allowed_ips2)
+        wazuh_log_monitor.start(timeout=5, callback=log_callback, error_message="Wazuh remoted didn't start as expected.")
```

Comment:
Remove hardcoded timeouts, use REMOTED_GLOBAL_TIMEOUT in remote module

```suggestion
        wazuh_log_monitor.start(timeout=REMOTED_GLOBAL_TIMEOUT, callback=log_callback, error_message='Wazuh remoted didn't start as expected.')
```